docs/data/date-pickers/date-range-picker/FormPropsDateRangePickers.tsx

```diff
@@ -1,41 +1,20 @@
 import * as React from 'react';
-import Typography from '@mui/material/Typography';
-import Box from '@mui/material/Box';
-import Stack from '@mui/material/Stack';
+import { DemoContainer, DemoItem } from 'docsx/src/modules/components/DemoContainer';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { Unstable_NextDateRangePicker as NextDateRangePicker } from '@mui/x-date-pickers-pro/NextDateRangePicker';
 
-function GridItem({
-  label,
-  children,
-  spacing = 1,
-}: {
-  label: string;
-  children: React.ReactNode;
-  spacing?: number;
-}) {
-  return (
-    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
-      <Typography variant="body2" sx={{ mb: spacing }}>
-        {label}
-      </Typography>
-      {children}
-    </Box>
-  );
-}
-
 export default function FormPropsDateRangePickers() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <Stack spacing={4}>
-        <GridItem label="disabled" spacing={2}>
+      <DemoContainer>
```

Comment:
I'm no longer gonna take screenshots, just again mentioning that this has changed in layout.

Reply:
[Same solution as ](https://github.com/mui/mui-x/pull/7149#discussion_r1044475136)